app/routines/add_account_to_salesforce.rb

```diff
@@ -0,0 +1,27 @@
+class AddAccountToSalesforce
+
+  lev_routine active_job_enqueue_options: { queue: :salesforce }
+
+  def exec(user:)
+      sf_ox_account = OpenStax::Salesforce::Remote::OpenStaxAccount.find_or_initialize_by(
+        account_uuid: user.uuid,
+      )
+
+      sf_ox_account.account_id = user.id
+      sf_ox_account.account_role = user.role.capitalize,
+      sf_ox_account.salesforce_contact_id = user&.salesforce_contact_id,
+      sf_ox_account.salesforce_lead_id = user&.salesforce_lead_id,
+      sf_ox_account.signup_date = user.created_at.strftime("%Y-%m-%dT%H:%M:%S%z")
+      sf_ox_account.account_environment = Rails.application.secrets.environment_name
+
+      if sf_ox_account.save!
+        user.salesforce_ox_account_id = sf_ox_account.id
+      else
```

Comment:
It's not a real ActiveRecord object so I'm not 100% sure but I think `save!` will throw so the else will never run

Reply:
fixed - and moved this to the (seemingly) more standard way of running background job - using the 'jobs' folder